__tests__/endpoints.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index')
+const app = require('../app');
+
+beforeEach(() => {
+    return seed({articleData, commentData, topicData, userData})
+})
+
+afterAll(() => {
+    return db.end()
+})
+describe('/api/topics', () => {
+    test('should respond with a status 200', () => {
+        return request(app).get('/api/topics').expect(200)
+        })
+    });
+    test('should respond with a slug and description', () => {
+        return request(app).get('/api/topics').expect(200).then((response) => {
+            expect(response.body).toEqual(
+                [
+                    {
+                      description: 'The man, the Mitch, the legend',
+                      slug: 'mitch'
+                    },
+                    {
+                      description: 'Not dogs',
+                      slug: 'cats'
+                    },
+                    {
+                      description: 'what books are made of',
+                      slug: 'paper'
+                    }
+                  ]
+            )
```

Comment:
Good work, while this works, there is a better way to be testing arrays: 

This test is too brittle, If any properties are added to each object, you will have to make lots of changes to the test. It is better practice to make the _same_ broad assertion _for each_ (hint) element of the array. That way if in future you add a column in the database you only have to change one assertion. Don't forget to include a length expectation as well!

Reply:
You should also add a generic 404 invalid endpoint test, you might find express handles this automatically.